postcss.config.js

```diff
@@ -1,4 +1,5 @@
-module.exports = {
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {
   plugins: {
```

Comment:
Had to change this to es module syntax since Vitest was complaining

Reply:
Can the export be named instead of turning off the eslint rule? We do the same thing in `next.config.mjs` file.

Reply:
Yep this can be changed